client/utils/networking.ts

```diff
@@ -5,3 +5,10 @@ const API_BASE_URL = process.env.API_SERVER_URL;
 export const getAllArtworks = (userId: number) => {
     return axios.get(`${API_BASE_URL}/users/${userId}/artworks`);
 };
+
+export const signIn = (code: string, strategy: string) => {
+    return axios.post(`http://localhost:3001/api/auth/signIn`, JSON.stringify({ code, strategy }), {
```

Comment:
여기는 `${API_BASE_URL} 적용이 안됬네
```suggestion
    return axios.post(`${API_BASE_URL}/api/auth/signIn`, JSON.stringify({ code, strategy }), {
```

Reply:
이 부분은 위 주의사항에 써놨는데, 여기서만 적용이 안되😂...